common/ast/src/parser/token.rs

```diff
@@ -878,7 +880,7 @@ impl TokenKind {
             | TokenKind::WHEN => true,
             | TokenKind::ARRAY
             | TokenKind::AS
-            // | TokenKind::CHAR
+            | TokenKind::CHAR => false,
```

Comment:
we have already a function named char(), so that return false in there.@[andylokandy](https://github.com/andylokandy)

Reply:
we can simply remove this line. Or else `TokenKind::{ARRAY, AS}` will mistakenly returns `false`.

Reply:
@TCeason this is a bug

Reply:
> @TCeason this is a bug

You mean this will cause apply and as return false?